spec/lib/life_game/cell_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../../spec_helper'
+
+RSpec.describe Cell do
+  describe '#alive?' do
+    subject { cell.alive? }
+    context "when State is DEAD" do
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.